src/foam/nanos/dig/DigWebAgent.java

```diff
@@ -213,6 +213,66 @@ public void execute(X x) {
           outputException(x, resp, format, out, error);
 
           return;
+        } else if (Format.JSONJ == format ) {
+          String dataJson = "[";
+          String dataJsonJ[] = data.split("\\s*\\r?\\n");
+          for (String i:dataJsonJ){
+            if (i.startsWith("p(") && i.endsWith(")")) {
```

Comment:
Also, you shouldn't need to do this. Have a look in the 'replay' of the JDAO as it has logic to apply the json/j operations, unless of course it does the same thing.